metaflow/includefile.py

```diff
@@ -282,8 +233,9 @@ class IncludeFile(Parameter):
     ----------
     name : str
         User-visible parameter name.
-    default : str
-        Default path to a local file.
+    default : str or a function
```

Comment:
We can consider limiting the scope of IncludeFiles to only explicit paths for now - not a huge deal though but good to control the sprawl of this functionality.

Reply:
It's pretty cheap. I think that if I explain in the code below properly, it shouldn't be hard to understand and the sprawl is limited to just those lines (everything else is the same as before for functions).